webapp/src/services/util.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import {GET_SUB_CATEGORIES, GET_CATEGORIES} from '../api/util';
+import SubCategory from '../model/SubCategory';
+import Category from '../model/Category';
+
+export function getSubCategories() {
+    return axios
+            .get(GET_SUB_CATEGORIES.url)
```

Comment:
why did you create another file for URL and request methods,
there's no use for them there!